### DISCUSSION TITLE
Consistent Hashing | System Design

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#54] [Comments]
# Consistent Hashing | System Design

In this tutorial, we will discuss Consistent Hashing

https://www.karanpratapsingh.com/courses/system-design/consistent-hashing

**Comment by bfdes** [comment]
AWS DynamoDB does not use consistent hashing. Consistent hashing is used in Amazon Dynamo, a different product. I know it is also a request routing option in distributed Memcached.
